### PR TITLE
Add persistent navigation bar to webapp

### PR DESCRIPTION
**Pull request by matthewfollegot**
This PR adds a persistent and responsive navigation bar to the application that can be viewed on each respective view that has been implemented.

- [x] pushed to Heroku
- [x] persistent nav bar
- [x] contains key tabs for searching users and viewing my profile
- [x] integrated Nicolas' changes for the profile page and added the Nav to that view as well

*Note to reviewer: squash commits upon merge*

### CODE REVIEW

**Comment by matthewfollegot** [comment]
[NavBar persisting above the webapp/Global CSS [3 SP]](https://trello.com/c/4NCianlH/41-navbar-persisting-above-the-webapp-global-css-3-sp)

**Comment by nicolasabousawan** [comment]
This ticket should include default css for the entire react app... I don't see where those changes are. Global fonts and sizing should be set up by this ticket so we have some sort of baseline for all components. 
Did I miss this somewhere?

**Review by nicolasabousawan — CHANGES_REQUESTED** [review]
Missing Global CSS features?  See previous comment

**Comment by matthewfollegot** [comment]
> This ticket should include default css for the entire react app... I don't see where those changes are. Global fonts and sizing should be set up by this ticket so we have some sort of baseline for all components.
> Did I miss this somewhere?

Sorry, this wasn't explicitly mentioned in the ticket's AC and I just so happened to leave the sprint planning early so I didn't have context. I'll add it explicitly to the AC and work on that.

**Comment by nicolasabousawan** [comment]
image

Very Nice

File: ui/README.md [hunk 1]
```diff
@@ -26,43 +27,3 @@ The build is minified and the filenames include the hashes.<br />
 Your app is ready to be deployed!
 
 See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
-
-### `npm run eject`
```
Comment (nicolasabousawan) [inline ui/README.md:30]: Why did we delete this? Do we no longer need it?

File: ui/src/App.js [hunk 2]
```diff
@@ -18,12 +18,11 @@ function App() {
         <Route exact path="/">
           <HomePage />
         </Route>
-        <Route exact path="/createProfile">
+        <Route exact path="/profiles/create">
```
Comment (nicolasabousawan) [inline ui/src/App.js:21]: Love the RESTful set up ;)

File: ui/src/Components/ProfilePage.jsx [hunk 3]
```diff
@@ -17,40 +29,52 @@ export default class ProfilePage extends React.Component {
...
-          </div>
-        )}
-      </Form>
+      <div>
+        <NavBar />
+        <Form>
+          {this.state.loading === (
+            <div>Loading...</div>
+          )}
+          {!this.state.profile ? (
```
Comment (nicolasabousawan) [inline ui/src/Components/ProfilePage.jsx:44]: Good catch separating the 2 states

File: ui/src/Components/ProfilePage.jsx [hunk 4]
```diff
@@ -17,40 +29,52 @@ export default class ProfilePage extends React.Component {
...
+                    <td>Phone Number</td>
+                    <td>{this.state.profile.result.contact_num}</td>
+                  </tr>
+                  <tr>
+                    <td>Extension</td>
+                    <td>{this.state.profile.result.contact_ext}</td>
+                  </tr>
+                </tbody>
+              </Table>
+              {/* <FormGroup row>
```
Comment (nicolasabousawan) [inline ui/src/Components/ProfilePage.jsx]: Is this supposed to be deleted?

**Review by nicolasabousawan — CHANGES_REQUESTED** [review]

File: ui/README.md (hunk 1 above)
Comment (nicolasabousawan) [reply ui/README.md:30]: spoke offline with  @matthewfollegot , this is indeed not needed

**Review by nicolasabousawan — APPROVED** [review]

**Pull request merged**